File: src/_as/00/math_clamp.ts
/** SSoT: {@link ../../ontology/math/math_clamp.md} */

@inline
export function math_clamp(val: i32, min: i32, max: i32): i32 {
if (val < min) return min;
if (val > max) return max;
return val;
}

File: src/00/sigma_core/src/ontology_gen/01/math_clamp.rs
// SSoT: file:///Users/s0fractal/OMEGA/I/math/math_clamp.md
#![allow(unused_imports)]
use super::super::L00::*;

pub fn math_clamp(val: i32, min: i32, max: i32) -> i32 {
    if val < min {
        min
    } else if val > max {
        max
    } else {
        val
    }
}
